workflows/treeval.nf

```diff
@@ -70,26 +70,33 @@ workflow TREEVAL {
 
     all_steps_list       = ["insilico_digest", "gene_alignment", "repeat_density", "gap_finder", "selfcomp", "synteny", "read_coverage", "telo_finder", "busco", "kmer", "hic_mapping", "NONE"]
 
-    jbrowse_exclude_list = ["insilico_digest", "gene_alignment", "selfcomp", "synteny", "busco", "kmer"]
-    rapid_exclude_list = ["gene_alignment", "repeat_density", "gap_finder", "read_coverage", "telo_finder", "hic_mapping"]
-    rapid_tol_exclude_list = ["gene_alignment", "repeat_density", "gap_finder", "read_coverage", "telo_finder", "kmer",  "hic_mapping"]
+    jbrowse_include_list = ["insilico_digest", "gene_alignment", "selfcomp", "synteny", "busco", "kmer"]
+    rapid_include_list = ["repeat_density", "gap_finder", "read_coverage", "telo_finder", "hic_mapping", "kmer"]
+    rapid_include_list = ["repeat_density", "gap_finder", "read_coverage", "telo_finder", "hic_mapping", "kmer"]
+    // RAPID_TOL is the same as RAPID, however includes more logic in the HIC_MAPPING workflow
```

Comment:
This kind of stuff is more useful in user docs.

Reply:
Done